docs/admin/kubeadm.md

```diff
@@ -210,6 +210,8 @@ networking:
 secrets:
   givenToken: <token|string>
 ```
+In addition, If authorizationMode set to `ABAC`, you may need to write the config to `/etc/kubernetes/abac_policy.json`,
```

Comment:
"may need" is a weak form and implies that there may be situations where this is not true. My suggestion:

"In addition, if authorizationMode is set to `ABAC`, you should write the config to `/etc/kubernetes/abac_policy.json`.
However, if authorizationMode set to `Webhook`, you should write the config to `/etc/kubernetes/webhook_authz.conf`.

Question: Is abac_policy only written in JSON? Is there no YAML file version of that?